src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -174,6 +182,11 @@ class RtpReceiverImpl @JvmOverloads constructor(
 
     init {
         logger.cdebug { "using executor ${executor.hashCode()}" }
+
+        if (lossFractionToIntroduce > 0) {
+            logger.warn("Will simulate ${lossFractionToIntroduce * 100}% packet loss.")
```

Comment:
Should these log messages distinguish receive-side and send-side loss?

Reply:
I think that's clear from the logger context:
```
2020-09-21 19:27:58.855 WARNING: [23] [confId=5c1f23210620082d epId=227 gid=-1] RtpSenderImpl.<init>#138: Will simulate 20.0% packet loss.
2020-09-21 19:27:58.856 WARNING: [23] [confId=5c1f23210620082d epId=227 gid=-1] RtpReceiverImpl.<init>#187: Will simulate 20.0% packet loss.
```